Makefile

```diff
@@ -6,11 +6,23 @@
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+init: ## Initialize the project
+	make install
+	make build
+
 install: ## Install the dependencies
-	npm install
+	cd curator && npm install
+	cd website && npm install
```

Comment:
**suggestion:** Have a look at [NPM workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces), this will allow you ton run a single install instead of one per folder